torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -779,9 +779,13 @@ void runNondiffOptimization(
   // Fuse the dequant - op - quant patterns into quantized ops
   QuantFusion(graph);
 
-  if (tensorExprFuserEnabled()) {
```

Comment:
Might be better to leave the old version of the check to make sure that the TE fuser is on (since we're explicitly flipping the flag in tensorexpr_fuser.cpp)